pkg/mgrconfig/config.go

```diff
@@ -112,12 +112,17 @@ type Config struct {
 	MaxCrashLogs int `json:"max_crash_logs"`
 
 	// Type of sandbox to use during fuzzing:
-	// "none": don't do anything special beyond resource sandboxing, default
-	// "setuid": impersonate into user nobody (65534). Supported only for some OSes.
-	// "namespace": create a new namespace for fuzzer using CLONE_NEWNS/CLONE_NEWNET/CLONE_NEWPID/etc,
-	//	requires building kernel with CONFIG_NAMESPACES, CONFIG_UTS_NS, CONFIG_USER_NS,
-	//	CONFIG_PID_NS and CONFIG_NET_NS. Supported only for some OSes.
-	// "android": (Android) Emulate permissions of an untrusted app.
+	// "none": test under root;
+	//      don't do anything special beyond resource sandboxing,
+	//      gives the most coverage, default
+	// "namespace": create a new user namespace for testing using CLONE_NEWUSER (supported only on Linux),
```

Comment:
It's not just `CLONE_NEWUSER`, but also `CLONE_NEWPID` and `CLONE_NEWNET`. Or is it not relevant?

Reply:
CLONE_NEWPID/NEWNET are done by all sandbox modes now and are more part of "resource sandboxing" rather than "privilege sandboxing".